docs/changehistory/NextVersion.md

```diff
@@ -3,6 +3,10 @@ publish: false
 ---
 # NextVersion
 
+## New Settings UI Features
+
+The @bentley/ui-core package has added the [SettingsManager]($ui-core) class that allow any number of [SettingsProvider]($ui-core) classes to be registered. These providers provide [SettingsTabEntry]($ui-core) definitions used to populate the [SettingsContainer]($ui-core) UI component with setting pages used to managed application settings. These new classes are marked as beta in this release and are subject to minor modifications in future releases.
```

Comment:
... that allows any ...
... used to manage application settings ...

Reply:
fixed